dags/solutions_team/configs/tensorflow/solutionsteam_tf_release_supported_config.py

```diff
@@ -26,11 +26,11 @@
 
 
 MAJOR_VERSION = "2"
-MINOR_VERSION = "17"
+MINOR_VERSION = "18"
 PATCH_VERSION = "0"
-RELEASE_CANDIDATE = "rc1"
-LIBTPU_VERSION = "1.11.0"
-KERAS_VERSION = "2.17.0rc0"
+RELEASE_CANDIDATE = "rc2"
+LIBTPU_VERSION = "1.12.0"
+KERAS_VERSION = "2.18.0rc0"
 MODELS_BRANCH = None  # "r2.17.0"
```

Comment:
I used model branch = "r2.18.0" for the tf 2.18 tests, not sure if we also need it here

Reply:
Oh Thanks WenXin, I think we need to use 2.18.0 if TF models already cut the release branch.